import TurbiniaException to partitions.py

```
Traceback (most recent call last):
  File "PATH/v2/lib/python3.8/site-packages/turbinia/workers/__init__.py", line 916, in run_wrapper
    self.result = self.run(evidence, self.result)
  File "PATH/v2/lib/python3.8/site-packages/turbinia/workers/partitions.py", line 144, in run
    path_specs = partitions.Enumerate(evidence)
  File "/PATH/v2/lib/python3.8/site-packages/turbinia/processors/partitions.py", line 49, in Enumerate
    raise TurbiniaException(
NameError: name 'TurbiniaException' is not defined

2021-03-05 18:45:56 [ERROR] PartitionEnumerationTask Task failed with exception: [name 'TurbiniaException' is not defined]
```

--- a/turbinia/processors/partitions.py
+++ b/turbinia/processors/partitions.py
@@ -21,6 +21,7 @@ from dfvfs.lib import definitions as dfvfs_definitions
 from dfvfs.lib import errors as dfvfs_errors
 
 from turbinia.lib.dfvfs_classes import UnattendedVolumeScannerMediator
+from turbinia import TurbiniaException
 
 log = logging.getLogger('turbinia')
 

